source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss

```diff
@@ -18,8 +18,8 @@ $card-border-width: rem-calc(1px);
 
 // Spacing
 $spacers: (
```

Comment:
Non-blocking. A complete aside, I have some questions on how we should be using spacers on projects. I'll ping you tomorrow.

Reply:
Really want to see us using the definitions in Bootstrap if a Bootstrap project. `$spacers` is left over from the early days of pattern lab starter.

Reply:
Awesome. I’ll need to check how to do this in Bootstrap. It’s inconsistent on my current project and I’d like to clear it up. I’ll reach out tomorrow.